### PR TITLE
[WFLY-13392] WFSM000001: Permission check failed ... FilePermission when Security Manager enabled and Web App tries to forward to jsp

### PR DESCRIPTION
**Pull request by gaol**
Issue: https://issues.redhat.com/browse/WFLY-13392

This is the test for https://issues.redhat.com/browse/UNDERTOW-1703, which should pass when undertow gets upgraded(https://issues.redhat.com/browse/WFCORE-4942)

### CODE REVIEW

File: testsuite/integration/secman/src/test/java/org/jboss/as/testsuite/integration/secman/ForwardJSPTestCase.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014, Red Hat, Inc., and individual contributors
```
Comment (darranl) [inline testsuite/integration/secman/src/test/java/org/jboss/as/testsuite/integration/secman/ForwardJSPTestCase.java:3]: FYI it is now 2020

Comment (gaol) [reply testsuite/integration/secman/src/test/java/org/jboss/as/testsuite/integration/secman/ForwardJSPTestCase.java]: Ah, yes. time flies :) 

Thank you for the review, updated to 2020. :)

**Comment by gaol** [comment]
retest this please

**Review by darranl — DISMISSED** [review]
Can we also please have a meaningful commit message and title to the pull request.

When coming back in 5 years time it is frustrating to look through the history of a class to understand the intent to be confronted with a list of "Test for WFLY-111", "Fix for AS7-654", "Update for WFL-222".

The commit message just needs to be the Jira issue plus a short sentence which is a summary of the change.  The title of the PR should also be updated so we have a better idea of what this covers so we know who to ask for review.

**Comment by gaol** [comment]
@darranl Updated as required. Thanks :)

**Comment by ivassile** [comment]
@darranl This change is scheduled for EAP 7.3.2. Can you please merge it upstream? Thanks!

**Review by darranl — APPROVED** [review]

**Comment by darranl** [comment]
@ivassile I am not planning to be merging today but have added the approval so upstream is ready to go.

**Pull request merged**